packages/manager/apps/pci-public-ip/src/pages/order/constants.ts

```diff
@@ -1,19 +1,35 @@
-export const URLS = new Map([
-  ['DE', 'https://www.ovh.de/order/express/#/express/review'],
-  ['EN', 'https://www.ovh.co.uk/order/express/#/express/review'],
-  ['ES', 'https://www.ovh.es/order/express/#/express/review'],
-  ['FR', 'https://www.ovh.com/fr/order/express/#/express/review'],
-  ['GB', 'https://www.ovh.co.uk/order/express/#/express/review'],
-  ['IE', 'https://www.ovh.ie/order/express/#/express/review'],
-  ['IT', 'https://www.ovh.it/order/express/#/express/review'],
-  ['MA', 'https://www.ovh.com/ma/order/express/#/express/review'],
-  ['NL', 'https://www.ovh.nl/order/express/#/express/review'],
-  ['PL', 'https://www.ovh.pl/order/express/#/express/review'],
-  ['PT', 'https://www.ovh.pt/order/express/#/express/review'],
-  ['SN', 'https://www.ovh.sn/order/express/#/express/review'],
-  ['TN', 'https://www.ovh.com/tn/order/express/#/express/review'],
-  ['DEFAULT', 'https://www.ovh.ie/order/express/#/express/review'],
-]);
+export const ORDER_URLS = {
+  EU: {
+    DE: 'https://www.ovh.de/order/express/#/express/review',
+    EN: 'https://www.ovh.co.uk/order/express/#/express/review',
+    ES: 'https://www.ovh.es/order/express/#/express/review',
+    FR: 'https://www.ovh.com/fr/order/express/#/express/review',
+    GB: 'https://www.ovh.co.uk/order/express/#/express/review',
+    IE: 'https://www.ovh.ie/order/express/#/express/review',
+    IT: 'https://www.ovh.it/order/express/#/express/review',
+    MA: 'https://www.ovh.com/ma/order/express/#/express/review',
+    NL: 'https://www.ovh.nl/order/express/#/express/review',
+    PL: 'https://www.ovh.pl/order/express/#/express/review',
+    PT: 'https://www.ovh.pt/order/express/#/express/review',
+    SN: 'https://www.ovh.sn/order/express/#/express/review',
+    TN: 'https://www.ovh.com/tn/order/express/#/express/review',
+    DEFAULT: 'https://www.ovh.ie/order/express/#/express/review',
+  },
+  CA: {
+    ASIA: 'https://ca.ovh.com/asia/order/express/#/express/review',
+    IN: 'https://ca.ovh.com/in/order/express/#/express/review',
+    AU: 'https://ca.ovh.com/au/order/express/#/express/review',
+    CA: 'https://ca.ovh.com/en/order/express/#/express/review',
+    QC: 'https://ca.ovh.com/fr/order/express/#/express/review',
+    SG: 'https://ca.ovh.com/sg/order/express/#/express/review',
+    WE: 'https://us.ovh.com/us/order/express/#/express/review',
+    WS: 'https://us.ovh.com/es/order/express/#/express/review',
+    DEFAULT: 'https://us.ovh.com/us/order/express/#/express/review',
+  },
+  US: {
+    DEFAULT: 'https://us.ovhcloud.com/order/express/#/express/review',
+  },
+};
```

Comment:
can't we put these in the package `packages/manager/modules/order` ?
To have a shared place for order urls and utils

Reply:
sure i just updated the PR